ManagedFeedImage+CoreDataClass.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  ManagedFeedImage+CoreDataClass.swift
+//  FeedStoreChallenge
+//
+//  Created by Mher Movsisyan on 28.08.21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+//
+
+import Foundation
+import CoreData
+
+@objc(ManagedFeedImage)
+class ManagedFeedImage: NSManagedObject {
```

Comment:
This class is not intended to be inherited thus it should be marked as `final`.
```suggestion
final class ManagedFeedImage: NSManagedObject {
```